### PR TITLE
feat: tvl metrics - highly addictive

### PR DESCRIPTION
**Pull request by albertocuestacanada**
`npx truffle exec --network mainnet scripts/pool_liquidity.js`

### CODE REVIEW

**Review by gakonst — APPROVED** [review]

**Review by aniemerg — APPROVED** [review]

File: scripts/pool_liquidity.js [hunk 1]
```diff
@@ -12,17 +23,38 @@ module.exports = async (deployer, network) => {
...
       const pool = await Pool.at(await migrations.contracts(web3.utils.fromAscii(contractName)))
       console.log(contractName)
-      console.log(`Dai Reserves:   ${await pool.getDaiReserves()}`)
-      console.log(`FYDai Reserves: ${await pool.getFYDaiReserves()}`)
+      const dai = await pool.getDaiReserves()
+      const fyDai = await pool.getFYDaiReserves()
+      console.log(`Dai Reserves:   ${dai.div(WAD)}`)
+      console.log(`FYDai Reserves: ${fyDai.div(WAD)}`)
       console.log()
+      tvl = tvl.add(dai).add(fyDai)
```
Comment (aniemerg) [inline scripts/pool_liquidity.js:54]: You can't add fyDai to the TVL. 1.) It's not external value, and 2.) Most of the fyDai are virtual!

**Review by aniemerg — CHANGES_REQUESTED** [review]
fyDai should not be added to TVL

File: scripts/pool_liquidity.js (hunk 1 above)
Comment (albertocuestacanada) [reply scripts/pool_liquidity.js]: Fixed now, thanks!

**Comment by albertocuestacanada** [comment]
This could work much faster by not checking migrations, but I'm struggling so for now let's use it as is.

**Pull request closed without merging**